2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,56 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+function RecipeCard(title, servings, ingredients) {
+  (this.title = title),
+    (this.servings = servings),
+    (this.ingredients = ingredients)
+   
+}
+
+/*1*/ let recipeOne  = new RecipeCard("Biriyani", 6, [ "meat", "spices", "oil",]);
+/*2*/ let recipeTwo = new RecipeCard("VegetableCurry", 4, ["vegetables", "turmeric", "onion"]);
+/*3*/ let recipeThree = new RecipeCard("AluVorta", 4, ["potato", "musturd oil", "chillies"]);
+/*4*/ let recipeFour = new RecipeCard("Payesh", 2, ["dried rice", "sugar", "milk"]);
+/*5*/ let recipeFive = new RecipeCard("TalerPitha", 5, ["plusm", "sugar", "milk"]);
+ 
+
+console.log(`${recipeOne.title}`)
+console.log(`Serves: ${recipeOne.servings}`)
+console.log(`Ingredients: 
+${recipeOne.ingredients[0]}
+${recipeOne.ingredients[1]}
+${recipeOne.ingredients[2]}`);
```

Comment:
well done, your console logs are formatted as as shown in the example and you're using dot notation well to access object values :)  Instead of having to type out ingredients[0] ingredients[1] ingredients[2], could you think of another way of approaching this? If your recipe ingredients increased to a larger number, having to type them out each time could get more problematic. (also really tasty sounding recipes btw!)